collectors/SDRSPropertiesCollector.py

```diff
@@ -47,7 +45,7 @@ def config_sdrsconfig_vmStorageAntiAffinityRules(self, metric_value):
                     str(rule.get('valid')).lower(),
                     vm
                     ])
-        return rule_labels, rule_label_values, len(mapped_vms)
+        return rule_labels, rule_label_values, 1
```

Comment:
`1` is static, because this is no _real_ value here?

Reply:
At the end it should be `sum` -able by promql. `sum by (rule_name)` should lead to 3.

Reply:
ok